cmd/configschema/otto/otto/server.go

```diff
@@ -0,0 +1,90 @@
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//       http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package otto
+
+import (
+	"net/http"
+
+	"go.opentelemetry.io/collector/component"
+
+	"github.com/open-telemetry/opentelemetry-collector-contrib/cmd/configschema"
+	"github.com/open-telemetry/opentelemetry-collector-contrib/internal/components"
+)
+
+func Server() {
+	mux := http.NewServeMux()
+
+	mux.Handle("/", http.FileServer(http.Dir("static")))
+
+	factories, err := components.Components()
+	if err != nil {
+		panic(err)
+	}
+
+	mux.Handle("/components", componentHandler{factories: factories})
+
+	ottoPipeline := &pipeline{
+		dr:        configschema.NewDirResolver("../../..", configschema.DefaultModule),
```

Comment:
this is probably not going to be accepted as is, as this allows to cd into a relative parent folder

Reply:
This points to the root directory of the collector source. I have made it configurable via cli arg.